react/__mocks__/mockSummaries.tsx

```diff
@@ -0,0 +1,64 @@
+import { Address } from 'vtex.checkout-graphql'
+import { CountryDescription } from '../mocks'
+
+export const mockSummaries: CountryDescription[] = [
+  [
+    [{ name: 'street' }, { delimiter: ' ', name: 'number' }],
+    [{ name: 'complement' }],
+    [{ name: 'postalCode' }],
+    [{ name: 'city' }, { delimiter: ', ', name: 'state' }],
+  ],
+  [
+    [
+      { name: 'street' },
+      { delimiter: ' ', name: 'number' },
+      { delimiter: ', ', name: 'complement' },
+    ],
+    [
+      { name: 'neighborhood', delimiterAfter: ' - ' },
+      { name: 'city' },
+      { delimiter: ' - ', name: 'state' },
+    ],
+    [{ name: 'postalCode' }],
+  ],
+  [
+    [{ name: 'street' }, { delimiter: ', ', name: 'complement' }],
+    [
+      { name: 'city' },
+      { delimiter: ', ', name: 'state' },
+      { delimiter: ' ', name: 'postalCode' },
+    ],
+  ],
+]
```

Comment:
Importing the values you could give names to each entry to facilitate your tests:
```suggestion
export const mockSummaries = {
  ARG: countryDescriptions[0],
  BRA: countryDescriptions[1],
  KOR: countryDescriptions[2],
}
```